config/default.yml

```diff
@@ -503,6 +503,16 @@ RSpec/ImplicitSubject:
   VersionChanged: '2.13'
   Reference: https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/ImplicitSubject
 
+RSpec/IndexedLet:
+  Description: Do not set up test data using indexes (e.g., `item_1`, `item_2`).
+  Enabled: pending
+  VersionAdded: "<<next>>"
+  Reference: https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/IndexedLet
+  MaxRepeats: 3
+  LetTypes:
+    - let
+    - let!
```

Comment:
We have `Helpers` [here](https://github.com/rubocop/rubocop-rspec/blob/39a0d090991e9bc96b297655dd808a41a02e1788/config/default.yml#L71), and it can be configured by users and [third-party gems](https://docs.rubocop.org/rubocop-rspec/third_party_rspec_syntax_extensions.html), e.g. [for `let_it_be`](https://github.com/test-prof/test-prof/pull/199). Wouldn't it be sufficient?